API/appsettingsapi.Development.json

```diff
@@ -32,13 +32,20 @@
             "Gitlab": {
                 "ClientId": "",
                 "ClientSecret": "",
-                "RedirectUri": "https://localhost:5001/api/wizard/oauth/callback/66de59d4-5db0-4bf8-a9a5-06abe8d3443a"
+                "RedirectUri": "https://localhost:5001/api/wizard/oauth/callback/66de59d4-5db0-4bf8-a9a5-06abe8d3443a",
+                "OauthUrl": "https://gitlab.com/oauth/authorize?client_id={0}&redirect_uri={1}&response_type=code&scope=read_repository+read_user"
+            },
+            "FontysGitlab": {
+                "ClientId": "536f6b353a9ecea950f69e9141ce87579376c279d7201c42099266a768148632",
+                "ClientSecret": "627a2ce856d47c95878e1dce2ada57ec135d3021fe916b1590f4cee6ddae37d7",
+                "RedirectUri": "https://localhost:5001/api/wizard/oauth/callback/6a73f5ee-e7a5-4ef5-b874-0da0993d6f13",
+                "OauthUrl": "https://git.fhict.nl/oauth/authorize?client_id={0}&redirect_uri={1}&response_type=code&scope=read_repository+read_user"
```

Comment:
I think this should be removed, right? Or can it be used for development?

@RubenFricke we should add FHICT GitLab API keys to staging / production